pcmanfm/preferences.ui

```diff
@@ -162,7 +162,7 @@
             <item row="2" column="0" colspan="2">
              <widget class="QCheckBox" name="ctrlRightClick">
               <property name="text">
-               <string>Show folder context menu with Ctrl + right click</string>
+               <string>Show folder context menu with Ctrl+right click</string>
```

Comment:
Unnecessary changes to strings break translations. " + " is as valid as "+".

Reply:
What it does is fuzzify translations, and alerting translators to how it could be more consistent is in this case warranted.

Reply:
"Right click" is a compound word with a space in the middle. We don't have a compound word like "Ctrl+right click".

Reply:
Could you rephrase that? How I understood it doesn't appear valid to me.

Reply:
> Could you rephrase that?

"2 red apples" vs. "2red apples"

Reply:
So you are saying it should be "Ctrl+ right click"?